examples/image_assurance_policy/README.md

```diff
@@ -0,0 +1,67 @@
+```go
+package main
+
+import (
+	"fmt"
+	"github.com/dome9/dome9-sdk-go/dome9"
+	"github.com/dome9/dome9-sdk-go/services/imageassurance/imageassurance_policy"
+)
+
+func main() {
+	// Pass accessID, secretKey, rawUrl, or set environment variables
+	config, _ := dome9.NewConfig("Access-ID", "Secret-Key", "https://api.dome9.com/v2/")
+	srv := imageassurance_policy.New(config)
+
+	request := imageassurance_policy.ImageAssurancePolicyRequest{
+		TargetId:        "<ENV ID or OU ID>",
+		TargetType:      "<Environment/OrganizationalUnit>",
+		RulesetId:       "<RuleSetID>",
+		NotificationIds: []string{"<NOTIFICATION_IDS>"},
+		AdmissionControllerAction:          "<Prevention/Detection>",
+		AdmissionControlUnScannedAction:      "<Prevention/Detection>",
+	}
+
+	// CreateImageAssurance Policy
```

Comment:
-> 	// Create Image Assurance Policy (also split ImageAssurance to Image Assurance in other comments and logs)